src/strongvelope/tlvstore.h

```diff
@@ -48,20 +48,19 @@ class TlvParser
 protected:
     const StaticBuffer& mSource;
     size_t mOffset;
-    bool mLegacyMode;
 public:
-    TlvParser(const StaticBuffer& source, size_t offset, bool legacyMode = false)
-    :mSource(source), mOffset(offset), mLegacyMode(legacyMode){}
+    TlvParser(const StaticBuffer& source, size_t offset)
+    :mSource(source), mOffset(offset){}
     bool getRecord(TlvRecord& record)
     {
        if (mOffset == Buffer::kNotFound)
             return false;
-        size_t typeLen = mLegacyMode ? 2 : 1;
+        size_t typeLen = 1;
```

Comment:
Now `typeLen` is always `1`, since the `sizeof(uint8_t)` is 1 byte. Shouldn't we adjust the code accordingly, instead of allocating a variable just for the shake of assigning it to a constant?